### PR TITLE
fix(test): create-config test interfering with actual config

### PR DESCRIPTION
**Pull request by k-dovgan**
The reason why `.universum.py` unexpectedly disappeared.

### CODE REVIEW

**Review by i-keliukh — APPROVED** [review]

File: tests/test_create_config.py [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+import os
 import subprocess
 
 from .utils import python
 
 
-def test_create_config():
-    result = subprocess.run([python(), "-m", "universum", "init"], capture_output=True, check=True)
+def test_create_config(tmpdir):
+    my_env = os.environ.copy()
```
Comment (i-keliukh) [inline tests/test_create_config.py:9]: In my opinion, the following construct is cleaner:
```python
test_environment=dict(os.environ, PYTHONPATH=os.getcwd())
```
Also note the better variable name.

File: tests/test_create_config.py [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+    result = subprocess.run([python(), "-m", "universum", "init"],
+                            capture_output=True, check=True, cwd=tmpdir, env=my_env)
     new_command = ''
     for line in result.stdout.splitlines():
         if line.startswith(b'$ '):           # result.stdout is a byte string
             new_command = line.lstrip(b'$ ')
-    new_result = subprocess.run(new_command.split(), capture_output=True, check=True)
-    Path(".universum.py").unlink()
-    artifacts = Path("artifacts")
+    new_result = subprocess.run(new_command.split(), capture_output=True, check=True, cwd=tmpdir, env=my_env)
```
Comment (i-keliukh) [inline tests/test_create_config.py:17]: The parameters repeat twice.
Maybe it's worth saving them to dict and reusing the dict on each call.

File: tests/test_create_config.py (hunk 1 above)
Comment (o-andrieiev) [reply tests/test_create_config.py]: agree

File: tests/test_create_config.py (hunk 2 above)
Comment (o-andrieiev) [reply tests/test_create_config.py]: agree

File: tests/test_create_config.py [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
     for line in result.stdout.splitlines():
         if line.startswith(b'$ '):           # result.stdout is a byte string
             new_command = line.lstrip(b'$ ')
-    new_result = subprocess.run(new_command.split(), capture_output=True, check=True)
-    Path(".universum.py").unlink()
-    artifacts = Path("artifacts")
+    new_result = subprocess.run(new_command.split(), capture_output=True, check=True, cwd=tmpdir, env=test_environment)
+    Path(tmpdir, ".universum.py").unlink()
+    artifacts = Path(tmpdir, "artifacts")
     artifacts.joinpath("CONFIGS_DUMP.txt").unlink()
```
Comment (i-keliukh) [inline tests/test_create_config.py:19]: Since moving to the temp directory, do we still need to clean artifacts after test?

**Review by i-keliukh — APPROVED** [review]

**Pull request merged**